### ISSUE TITLE
MODULE NOT FOUND (Mac)

### ISSUE THREAD

**Issue by Seb137** [OstlerDev/PopcornTV#101]
Hi,
I get the following message when I'm trying to generate ATV certificate:
imac:PopcornTV maison$ sudo node atv.js
Password:
notice: Starting PopcornTV
DNS: DnsProxy binding on 192.168.1.18:53
error: There is an error starting Popcorn TV, please post this on the Github page
{ [Error: Cannot find module 'popcorn-streamer'] code: 'MODULE_NOT_FOUND' }
imac:PopcornTV maison$ 

What I'm doing wrong?
Thank you :)

**Comment by Seb137** [comment]
Since I red this post https://github.com/OstlerDev/PopcornTV/issues/24
This is npm install returned:
imac:~ maison$ cd Desktop/
imac:Desktop maison$ cd PopcornTV
imac:PopcornTV maison$ npm install
npm WARN package.json PopcornTV@0.1.6 No license field.
npm WARN locking Error: EACCES, open '/Users/maison/.npm/_locks/popcorn-streamer-99364dd8bcc364a7.lock'
npm WARN locking     at Error (native)
npm WARN locking  /Users/maison/.npm/_locks/popcorn-streamer-99364dd8bcc364a7.lock failed { [Error: EACCES, open '/Users/maison/.npm/_locks/popcorn-streamer-99364dd8bcc364a7.lock']
npm WARN locking   errno: -13,
npm WARN locking   code: 'EACCES',
npm WARN locking   path: '/Users/maison/.npm/_locks/popcorn-streamer-99364dd8bcc364a7.lock' }
npm ERR! Darwin 12.6.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.12.7
npm ERR! npm  v2.11.3

npm ERR! Attempt to unlock /Users/maison/Desktop/PopcornTV/node_modules/popcorn-streamer, which hasn't been locked
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/maison/Desktop/PopcornTV/npm-debug.log

**Comment by OstlerDev** [comment]
Try using `sudo npm install`

**Comment by Seb137** [comment]
thank you
I already did sudo npm install and I got this:
npm WARN package.json PopcornTV@0.1.6 No license field.
npm WARN addRemoteGit Error: Command failed: git config --get remote.origin.url
npm WARN addRemoteGit 
npm WARN addRemoteGit     at ChildProcess.exithandler (child_process.js:751:12)
npm WARN addRemoteGit     at ChildProcess.emit (events.js:110:17)
npm WARN addRemoteGit     at maybeClose (child_process.js:1015:16)
npm WARN addRemoteGit     at Socket.<anonymous> (child_process.js:1183:11)
npm WARN addRemoteGit     at Socket.emit (events.js:107:17)
npm WARN addRemoteGit     at Pipe.close (net.js:485:12)
npm WARN addRemoteGit  git+https://github.com/xeoncore/torrent-stream.git resetting remote /Users/maison/.npm/_git-remotes/https-github-com-xeoncore-torrent-stream-git-864d0b24 because of error: { [Error: Command failed: git config --get remote.origin.url
npm WARN addRemoteGit ]
npm WARN addRemoteGit   killed: false,
npm WARN addRemoteGit   code: 1,
npm WARN addRemoteGit   signal: null,
npm WARN addRemoteGit   cmd: 'git config --get remote.origin.url' }
npm WARN addRemoteGit Error: Command failed: git config --get remote.origin.url
npm WARN addRemoteGit 
npm WARN addRemoteGit     at ChildProcess.exithandler (child_process.js:751:12)
npm WARN addRemoteGit     at ChildProcess.emit (events.js:110:17)
npm WARN addRemoteGit     at maybeClose (child_process.js:1015:16)
npm WARN addRemoteGit     at Socket.<anonymous> (child_process.js:1183:11)
npm WARN addRemoteGit     at Socket.emit (events.js:107:17)
npm WARN addRemoteGit     at Pipe.close (net.js:485:12)
npm WARN addRemoteGit  git+https://github.com/xeoncore/progress-stream.git#7070db resetting remote /Users/maison/.npm/_git-remotes/https-github-com-xeoncore-progress-stream-git-609443b1 because of error: { [Error: Command failed: git config --get remote.origin.url
npm WARN addRemoteGit ]
npm WARN addRemoteGit   killed: false,
npm WARN addRemoteGit   code: 1,
npm WARN addRemoteGit   signal: null,
npm WARN addRemoteGit   cmd: 'git config --get remote.origin.url' }
npm ERR! git clone --template=/Users/maison/.npm/_git-remotes/_templates --mirror https://github.com/xeoncore/progress-stream.git /Users/maison/.npm/_git-remotes/https-github-com-xeoncore-progress-stream-git-609443b1: dyld: lazy symbol binding failed: Symbol not found: ___strlcpy_chk
npm ERR! git clone --template=/Users/maison/.npm/_git-remotes/_templates --mirror https://github.com/xeoncore/progress-stream.git /Users/maison/.npm/_git-remotes/https-github-com-xeoncore-progress-stream-git-609443b1:   Referenced from: /usr/local/git/bin/git
npm ERR! git clone --template=/Users/maison/.npm/_git-remotes/_templates --mirror https://github.com/xeoncore/progress-stream.git /Users/maison/.npm/_git-remotes/https-github-com-xeoncore-progress-stream-git-609443b1:   Expected in: /usr/lib/libSystem.B.dylib
npm ERR! git clone --template=/Users/maison/.npm/_git-remotes/_templates --mirror https://github.com/xeoncore/progress-stream.git /Users/maison/.npm/_git-remotes/https-github-com-xeoncore-progress-stream-git-609443b1: 
npm ERR! git clone --template=/Users/maison/.npm/_git-remotes/_templates --mirror https://github.com/xeoncore/progress-stream.git /Users/maison/.npm/_git-remotes/https-github-com-xeoncore-progress-stream-git-609443b1: dyld: Symbol not found: ___strlcpy_chk
npm ERR! git clone --template=/Users/maison/.npm/_git-remotes/_templates --mirror https://github.com/xeoncore/progress-stream.git /Users/maison/.npm/_git-remotes/https-github-com-xeoncore-progress-stream-git-609443b1:   Referenced from: /usr/local/git/bin/git
npm ERR! git clone --template=/Users/maison/.npm/_git-remotes/_templates --mirror https://github.com/xeoncore/progress-stream.git /Users/maison/.npm/_git-remotes/https-github-com-xeoncore-progress-stream-git-609443b1:   Expected in: /usr/lib/libSystem.B.dylib
npm ERR! git clone --template=/Users/maison/.npm/_git-remotes/_templates --mirror https://github.com/xeoncore/torrent-stream.git /Users/maison/.npm/_git-remotes/https-github-com-xeoncore-torrent-stream-git-864d0b24: dyld: lazy symbol binding failed: Symbol not found: ___strlcpy_chk
npm ERR! git clone --template=/Users/maison/.npm/_git-remotes/_templates --mirror https://github.com/xeoncore/torrent-stream.git /Users/maison/.npm/_git-remotes/https-github-com-xeoncore-torrent-stream-git-864d0b24:   Referenced from: /usr/local/git/bin/git
npm ERR! git clone --template=/Users/maison/.npm/_git-remotes/_templates --mirror https://github.com/xeoncore/torrent-stream.git /Users/maison/.npm/_git-remotes/https-github-com-xeoncore-torrent-stream-git-864d0b24:   Expected in: /usr/lib/libSystem.B.dylib
npm ERR! git clone --template=/Users/maison/.npm/_git-remotes/_templates --mirror https://github.com/xeoncore/torrent-stream.git /Users/maison/.npm/_git-remotes/https-github-com-xeoncore-torrent-stream-git-864d0b24: 
npm ERR! git clone --template=/Users/maison/.npm/_git-remotes/_templates --mirror https://github.com/xeoncore/torrent-stream.git /Users/maison/.npm/_git-remotes/https-github-com-xeoncore-torrent-stream-git-864d0b24: dyld: Symbol not found: ___strlcpy_chk
npm ERR! git clone --template=/Users/maison/.npm/_git-remotes/_templates --mirror https://github.com/xeoncore/torrent-stream.git /Users/maison/.npm/_git-remotes/https-github-com-xeoncore-torrent-stream-git-864d0b24:   Referenced from: /usr/local/git/bin/git
npm ERR! git clone --template=/Users/maison/.npm/_git-remotes/_templates --mirror https://github.com/xeoncore/torrent-stream.git /Users/maison/.npm/_git-remotes/https-github-com-xeoncore-torrent-stream-git-864d0b24:   Expected in: /usr/lib/libSystem.B.dylib
/
> images@2.1.6 install /Users/maison/Desktop/PopcornTV/node_modules/images
> echo "Hello, World."

Hello, World.
npm ERR! Darwin 12.6.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.12.7
npm ERR! npm  v2.11.3

npm ERR! Command failed: git clone --template=/Users/maison/.npm/_git-remotes/_templates --mirror https://github.com/xeoncore/progress-stream.git /Users/maison/.npm/_git-remotes/https-github-com-xeoncore-progress-stream-git-609443b1
npm ERR! dyld: lazy symbol binding failed: Symbol not found: ___strlcpy_chk
npm ERR!   Referenced from: /usr/local/git/bin/git
npm ERR!   Expected in: /usr/lib/libSystem.B.dylib
npm ERR! 
npm ERR! dyld: Symbol not found: ___strlcpy_chk
npm ERR!   Referenced from: /usr/local/git/bin/git
npm ERR!   Expected in: /usr/lib/libSystem.B.dylib
npm ERR! 
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/maison/Desktop/PopcornTV/npm-debug.log

**Comment by OstlerDev** [comment]
It could be an issue with Git on your computer. Try deleting the folder and starting again from scratch, maybe that will help?

**Comment by Seb137** [comment]
Hi OstlerDev,
thank you for answering.
I got progress and resolved the point. Here the steps I followed:

1) get install xCode
2) clone the repository from xCode
3) npm install succeed!
4) node atv.js succeed!

Now I get my terminal screen:
notice: Starting PopcornTV
DNS: DnsProxy binding on 192.168.1.18:53
Web: listening on 192.168.1.18:80
Web: SSL Web: listening on 192.168.1.18:443

When I add profile, the terminal writes down:
DNS: itunes.apple.com
DNS: init.itunes.apple.com
DNS: itunes-cdn.itunes-apple.com.akadns.net
DNS: xp.apple.com
DNS: configuration.apple.com
DNS: p01-keyvalueservice.icloud.com
DNS: itunes-cdn.itunes-apple.com.akadns.net
DNS: se.itunes.apple.com
DNS: bookkeeper.itunes.apple.com
DNS: init-cdn.itunes-apple.com.akadns.net
DNS: upp.itunes.apple.com
DNS: mzuserxp.itunes-apple.com.akadns.net
DNS: e5153.a.akamaiedge.net

I launched Popcorn app on the Mac
I set the ATV (DNS equals Mac's IP address, and profile http://trailers.apple.com/trailers.cer) but still blocked on add profile screen :(
So I did reboot, relaunch terminal with node atv.js command and run Popcorn app. I closed iTunes (who open up when computer starts).
I think I'm almost to the Graal but I'm surely missing steps.
Any ideas?

Thank you again for your support
